src/tools/genpolicy/rules.rego

```diff
@@ -587,7 +749,6 @@ allow_process_common(p_process, i_process, s_name, s_namespace) {
 allow_process(p_process, i_process, s_name, s_namespace) {
     print("allow_process: start")
 
-    allow_args(p_process, i_process, s_name)
```

Comment:
by no longer substituting env variables in args, the current way of validating args no longer applies